src/MLOps.NET/Storage/Deployments/SchemaGenerator.cs

```diff
@@ -0,0 +1,37 @@
+﻿using ICSharpCode.Decompiler;
+using ICSharpCode.Decompiler.CSharp;
+using ICSharpCode.Decompiler.TypeSystem;
+using System;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+
+namespace MLOps.NET.Storage.Deployments
+{
+    /// <summary>
+    /// 
+    /// </summary>
+    public sealed class SchemaGenerator
```

Comment:
```suggestion
    internal sealed class SchemaGenerator
```

Reply:
Could you also real quick just add an interface to this class and move the summary comments to the interface? After that I happy to approve and merge the PR

Reply:
E.g. under an `Interfaces` folder

```
namespace MLOps.NET.Storage.Deployments
{
    /// <summary>
    /// Auto-generates a class based on run-time instance
    /// </summary>

    public interface ISchemaGenerator 
    {
        /// <summary>
        /// Searches the entire app domain to return the class definition as a string, using ilSpy
        /// </summary>
        /// <typeparam name="T"></typeparam>
        /// <param name="className"></param>
        /// <returns>Run-time instance's class definition</returns>
        string GenerateDefinition<T>(string className) where T: class;
    }
}
```